src/momento/simple_cache_client.py

```diff
@@ -48,8 +50,22 @@
 class SimpleCacheClient:
     """Synchronous Simple Cache Client"""
 
+    _NUM_CLIENTS = 1
+    """(async client only) For high load, we might get better performance with multiple clients,
+    because the server is configured to allow a max of 100 streams per connection.
+`
+    In the javascript SDK, multiple clients resulted in an obvious performance improvement.
+    However, in the python SDK I have not yet been able to observe a clear benefit.
+    So for now, we are putting the plumbing in place so that we can more easily test performance
+    with multiple connections in the future, but we are leaving the default value set to 1.
+
+    We are hard-coding the value for now, because we haven't yet designed the API for
+    users to use to configure tunables:
+    https://github.com/momentohq/dev-eco-issue-tracker/issues/85
+    """
```

Comment:
Should this be user-visible documentation?

Reply:
The field is internal so it's OK.

Reply:
It won't be visible in generated docs?

Reply:
If that's acceptable doc style, ok.